dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -66,17 +66,50 @@ def stage_ic(self):
 
         return task
 
+    def prep_sfc(self):
+
+        dependencies = None
+        if self.options['do_prep_sfc']:
```

Comment:
This method should not be in the task list if this option is NO.
```suggestion
```
The remaining of the method will need to be left indented.

Reply:
Fixed, thanks!